### ISSUE TITLE
RCS1032 False Negative

### ISSUE THREAD

**Issue by nnpcYvIVl** [JosefPihrt/Roslynator#379]
Shouldn't these three lines be flagged by RCS1032?

```csharp
namespace RCS1032_False_Negative
{
    public class Class
    {
        private static void Method()
        {
            int x = (100);
            int y = (1 * 2);
            byte[] bytearray = new byte[(500)];
        }
    }
}
```

**Comment by JosefPihrt** [comment]
1) yes
2) Someone may prefer to keep more complex expression in parentheses so I would not add this one to RCS1032.
3) yes

**Comment by JosefPihrt** [comment]
fixed with ea7eb884bcad2feae1067a58e44c844f3c13cf8e
